server/app.ts

```diff
@@ -208,7 +248,11 @@ export default function createApp(userService: UserService): express.Application
 
   app.use(authorisationMiddleware())
   app.use('/', indexRoutes(standardRouter(userService)))
+
   app.use((req, res, next) => next(createError(404, 'Not found')))
+
+  // The error handler must be before any other error middleware and after all controllers
+  app.use(Sentry.Handlers.errorHandler())
```

Comment:
Is there also a way to notify Sentry about a caught error? Most errors will be caught rather than falling. through to this handler

Reply:
You'd have to push the caught error up manually, but yep can be done pretty simply: https://docs.sentry.io/platforms/node/usage/#capturing-errors

Reply:
great